apps/health_campaign_field_worker_app/lib/pages/beneficiary_registration/household_location.dart

```diff
@@ -104,22 +92,6 @@ class _HouseholdLocationPageState
                               isHeadOfHousehold,
                             ) {
                               var addressModel = AddressModel(
-                                addressLine1: addressLine1 != null &&
```

Comment:
Add boundary name in addressline1 as it was earlier

Reply:
I have reverted the change as requested